metrics.ini

```diff
@@ -2103,6 +2103,7 @@ LifePercentChangeHeld=SL.Metrics[SL.Global.GameMode]["LifePercentChangeHeld"]
 LifePercentChangeLetGo=SL.Metrics[SL.Global.GameMode]["LifePercentChangeLetGo"]
 
 [ScoreKeeperNormal]
+PercentScoreWeightCheckpointHit=SL.Metrics[SL.Global.GameMode]["PercentScoreWeightCheckpointHit"]

```

Comment:
Mention that this metric and GradeWrightCheckpointHit are only used for the pump game mode type.

Reply:
I added a comment in SL_Init.lua (where we set the values). The comment also explains why we set them to 0.

I remember you telling me this before, and I added a comment in my first branch, but I forgot to add it again when I re-created the branch. Sorry for that!